packages/cfd/src/Stores/Modules/CFD/Helpers/useCfdStores.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, PropsWithChildren, useContext } from 'react';
+import { useStore } from '@deriv/stores';
+import type { TCFDStore } from 'src/types/cfd-store.types';
+
+const CFDStoreContext = createContext<TCFDStore | null>(null);
+
+export const CFDStoreProvider = ({ children }: PropsWithChildren<unknown>) => {
+    const { modules } = useStore();
+
```

Comment:
```suggestion
    const { modules } = useStore();
    // const memoizedValue = useMemo(() => new CFDStore(), []);

```

Reply:
what's this comment is for ? `    // const memoizedValue = useMemo(() => new CFDStore(), []);`